contact.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Contact Us</title>
+  </head>
+  <body>
+    <header>
+      <ul>
+        <li><a href="index.html">Home<a></li>
+      </ul>
+    </header>
+    <h1>Contact Us</h1>
+    <h2>Mail Address:</h2>
+    <p>707 Broadway, Ste 800

```

Comment:
Not a big deal, but you probably don't want your actual home address here if that is what this is.